packages/ra-ui-materialui/src/input/common/BooksCreate.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {
+    DateInput,
+    NumberInput,
+    TextInput,
+    TimeInput,
+    Create,
+    SimpleForm,
+    TranslatableInputs,
+} from './../..';
+
+export const BooksCreate = () => (
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** ‏I'm not very found of including storybook only code in the `packages/ra-ui-materialui/src/input/common` folder. Added to that there is already a `packages/ra-ui-materialui/src/input/common.tsx` file, which will make it more confusing.

If you really need all these `FullApp` stories, then I recommend we move this folder to `packages/ra-ui-materialui/src/storybook/common` instead.

@djhi or @fzaninotto  any thoughts?